packages/dashboard/src/app/views/myStories/karma/myStoriesList.karma.js

```diff
@@ -82,6 +82,14 @@ describe('CUJ: Creator can view their stories in list view: ', () => {
 
       expect(listViewTable).toBeTruthy();
 
+      const { isLoading } = await getStoriesState();
+
+      await waitFor(() => {
+        if (isLoading) {
+          throw new Error('Loading list view');
+        }
+      });
```

Comment:
I think you'd need to pull in `isLoading` into the `waitFor` otherwise it is looking at a stale `isLoading` variable every time the callback passed to the `waitFor` is called
```js
 await waitFor(() => {
       const { isLoading } = await getStoriesState();
        if (isLoading) {
          throw new Error('Loading list view');
        }
 });
```

Reply:
Updated
https://github.com/GoogleForCreators/web-stories-wp/pull/10671/commits/8ee88bc7a1b5974adc3b13c4d6dfa95b200ff0fb#diff-a5157c7ecbd48941338c3d934f09203e6c7ab80e958e67362ee4c3a733b051fdR117

Same result
https://percy.io/google/web-stories-wp-dashboard/builds/16121346/unchanged/908252407?browser=chrome&viewLayout=side-by-side&viewMode=new&width=1600

Reply:
sorry for the delay here, i was out last week. 

I've pulled the changes down and am running locally and it's working as expected. definitely don't see the spinner all the time.

Reply:
ah, i know what's missing. 

The fixture's missing `allPagesFetched`. So it's actually working exactly how it should be given that the page thinks there's more stories to possibly load.

Reply:
> definitely don't see the spinner all the time.

Same 

Pushed one more update to capture the value of `isLoading` into the filename captured --- loading = false but the spinner is still there.

<img width="1258" alt="Screen Shot 2022-02-28 at 5 21 15 PM" src="https://user-images.githubusercontent.com/62242/156068340-b22f713e-36eb-47d0-bedc-ae44d050513f.png">

Reply:
did you see my other note below? The spinner is showing up because `allPagesFetched` isn't in the karma fixture so it's value isn't getting mocked out making it always false so the spinner is showing up because of that. isLoading isn't the problem.

Reply:
> did you see my other note below my initial reply? 

Just did

Reply:
So, `allPagesFetched` is our way of knowing when the "infinite scroll" should grab the next page of stories. That's why it's separate from `isLoading`. Templates, to the point on slack, have their own version of this and it's always true because templates are just imported data not an API. 

`allPagesFetched` needs to get added to the mocked story state, here: https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/dashboard/src/karma/apiProviderFixture.js#L109 

The highlighted line here is the only addition necessary. I tested locally and everything is still succeeding. 
<img width="330" alt="Screen Shot 2022-02-28 at 4 22 39 PM" src="https://user-images.githubusercontent.com/10720454/156075381-b6f657e2-5aba-4de4-ac62-55f36dc3bf83.png">


Just like here for templates: https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/dashboard/src/karma/apiProviderFixture.js#L256 

Making this be `true` won't hurt anything because there's only 1 page of stories in the fixture. 

`allPagesFetched` is getting set in the stories reducer, here: https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/dashboard/src/app/reducer/stories.js#L154 for reference.

Reply:
🎉 

https://percy.io/google/web-stories-wp-dashboard/builds/16126194/changed/908497842?browser=chrome&browser_ids=20&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1600&widths=1600

Reply:
Thanks for the help Brittany!
Looks like it‘s my bad for pointing at isLoading while allPagesFetched was mentioned initially.